chart/templates/_rbac.tpl

```diff
@@ -0,0 +1,139 @@
+{{- define "vcluster.clusterRoleName" -}}
+{{- printf "vc-%s-v-%s" .Release.Name .Release.Namespace | trunc 63 | trimSuffix "-" -}}
+{{- end -}}
+
+{{- define "vcluster.clusterRoleNameMultinamespace" -}}
+{{- printf "vc-mn-%s-v-%s" .Release.Name .Release.Namespace | trunc 63 | trimSuffix "-" -}}
+{{- end -}}
+
+{{/*
+  Whether to create a cluster role or not
+*/}}
+{{- define "vcluster.createClusterRole" -}}
+{{- if not .Values.rbac.clusterRole.disabled -}}
+{{- if or
+    .Values.rbac.clusterRole.overwriteRules
+    (not (empty (include "vcluster.rbac.clusterRoleExtraRules" . )))
+    (not (empty (include "vcluster.plugin.clusterRoleExtraRules" . )))
+    (not (empty (include "vcluster.generic.clusterRoleExtraRules" . )))
+    .Values.networking.replicateServices.fromHost
+    .Values.pro
```

Comment:
Didn't we want to refactor this, so that a pro one doesn't by default need a cluster role?

Reply:
This is legacy and won't be used for the new version anymore. We still need this for older loft versions though as they will set that field automatically in a controller.